post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -6,21 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. You can deploy a cluster with multi-architecture compute machines by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are only available on AWS or Azure installer-provisioned infrastructures and Bare Metal user-provisioned infrastructures with x86_64 control plane machines. 
```

Comment:
```suggestion
An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal user-provisioned infrastructures with x86_64 control plane machines. 
```

Reply:
Place "only" immediately before the word or phrase it modifies. Source: https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only

Reply:
You are missing backticks on the `x86_64` in the last sentence.